File: app/src/main/java/org/example/gameByBall2/Ex_A.java
package org.example.gameByBall2;

import processing.core.PApplet;

public class Ex_A {
  public static class First extends PApplet {

    int count = 10;

    public void settings() {
      size((this.count + 1) * 100, (this.count + 1) * 100);
    }

    public void draw() {
      for (int i = 0; i < this.count; ++i) {
        if (i % 3 == 0) {
          fill(color(0, 0, 255));
        } else {
          fill(color(255, 255, 255));
        }
        ellipse(i * 100 + 100, i * 100 + 100, 100, 100);
      }
    }
  }

  public static class Third extends PApplet {
    int width = 5;
    int height = 5;

    public void settings() {
      size((this.width + 1) * 100, (this.height + 1) * 100);
    }

    public void draw() {
      int count = 0;
      for (int x = 0; x < this.width; ++x) {
        for (int y = 0; y < this.height; ++y) {
          if (count % 7 == 0 && count % 3 == 0) {
            fill(color(0, 255, 255));
          } else if (count % 7 == 0) {
            fill(color(0, 0, 255));
          } else if (count % 3 == 0) {
            fill(color(0, 255, 0));
          } else {
            fill(color(255, 255, 255));
          }
          ellipse(x * 100 + 100, y * 100 + 100, 80, 80);
          ++count;
        }
      }
    }
  }

  public static void main() {
    PApplet.main(Ex_A.First.class.getName());
    PApplet.main(Ex_A.Third.class.getName());
  }
}

File: app/src/main/kotlin/org/example/gameByBall2/Ex_A.kt
package org.example.gameByBall2

import processing.core.PApplet

object ExA {
    class First : PApplet() {
        private val count = 10

        override fun settings() {
            size((count + 1) * 100, (count + 1) * 100)
        }

        override fun draw() {
            for (i in 0 until count) {
                if (i % 3 == 0) {
                    fill(color(0, 0, 255))
                } else {
                    fill(color(255, 255, 255))
                }
                ellipse(i * 100f + 100f, i * 100f + 100f, 100f, 100f)
            }
        }
    }

    class Third : PApplet() {
        private val width = 5
        private val height = 5

        override fun settings() {
            size((width + 1) * 100, (height + 1) * 100)
        }

        override fun draw() {
            var count = 0
            for (x in 0 until width) {
                for (y in 0 until height) {
                    when {
                        count % 7 == 0 && count % 3 == 0 -> fill(color(0, 255, 255))
                        count % 7 == 0 -> fill(color(0, 0, 255))
                        count % 3 == 0 -> fill(color(0, 255, 0))
                        else -> fill(color(255, 255, 255))
                    }
                    ellipse(x * 100f + 100f, y * 100f + 100f, 80f, 80f)
                    count++
                }
            }
        }
    }

    fun main() {
        PApplet.main(First::class.java.name)
        PApplet.main(Third::class.java.name)
    }
}
